app/views/home/home.html.erb

```diff
@@ -0,0 +1,20 @@
+<section id="content">
+  <ol id="links-list">
+    <li>
+      <span class="link">
+        <%= link_to "The coolest thing you've ever seen.", "#" %>
+      </span>  
+      <span class="details">
+        <p> 
+          posted by
+          <%= link_to "mr.man", "#" %> 
+          5 minutes ago
+        </p>
+      </span>
+    </li>
+  </ol>
+</section> 
+
+
+
+
```

Comment:
you might wanna configure your editor to trim up these extra lines and any trailing spaces for you automatically